zstackctl/zstackctl/ctl.py

```
@@ -4119,7 +4132,9 @@ def run(self, args):
         SpinnerInfo.spinner_status = reset_dict_value(SpinnerInfo.spinner_status,False)
         SpinnerInfo.spinner_status['backup_db'] = True
         ZstackSpinner(spinner_info)
-        (status, output) =  subprocess.getstatusoutput("zstack-ctl dump_mysql >> /dev/null 2>&1")
+        (status, output) =  subprocess.getstatusoutput("zstack-ctl dump_mysql")
```

Ruff (0.15.9)

[error] Starting a process with a shell: seems safe, but may be changed in the future; consider rewriting without `shell`

(S605)